app/controllers/films_controller.rb

```diff
@@ -0,0 +1,9 @@
+class FilmsController < ApplicationController
+  def index
+    @films = Film.all
+  end
+
+  def show
+    @film = Film.find(params[:id])
```

Comment:
what answer will the server give if i write invalid id?

Is this good server behavior?